enigma/src/main/java/org/quiltmc/enigma/source/Decompiler.java

```diff
@@ -4,10 +4,9 @@
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 public interface Decompiler {
-	@Deprecated // use remapper specific one for easy doc inclusion
-	default Source getSource(String className) {
-		return this.getSource(className, null);
+	default Source getUndocumentedSource(String className) {
+		return this.getUndocumentedSource(className, null);
 	}
 
-	Source getSource(String className, @Nullable EntryRemapper remapper);
+	Source getUndocumentedSource(String className, @Nullable EntryRemapper remapper);
```

Comment:
This one does produce documented source :p

Reply:
At least if remapper isn't null

Reply:
throwing up how did I not notice that happened :p

Reply:
ah intellij overload renaming
fixed!